stdlib/private/DifferentiationUnittest/DifferentiationUnittest.swift

```diff
@@ -172,24 +172,28 @@ extension Tracked : Differentiable where T : Differentiable, T == T.TangentVecto
 
 extension Tracked where T : Differentiable, T == T.TangentVector {
   @usableFromInline
```

Comment:
Note for later: We should make `@derivative` and `@transpose` imply `@usableFromInline` for `internal` declarations.